frontEnd/order_up/src/utils/api.ts

```diff
@@ -166,6 +166,22 @@ export async function login(username: String, password: String, signal?: AbortSi
     return await fetchJson(url, options);
 }
 
+export async function completeOrder(id: number, signal?: AbortSignal) {
+    const url = new URL(`${API_BASE_URL}/api/order/${id}`);
+    const body = { customerName: null,
+    			  userID: null,
+    			  status: "COMPLETED"}
```

Comment:
This looks great. I would remove the `customerName` and `userID` properties from the body rather than setting them to null. The backend will only modify the fields that you pass it(in this case, `status`).